src/libs/print_settings.c

```diff
@@ -360,6 +360,9 @@ static int _print_job_run(dt_job_t *job)
   dt_tag_new(tag, &tagid);
   dt_tag_attach_from_gui(tagid, params->imgid, FALSE, FALSE);
 
+  /* register print timestamp in cache */
+  image_cache_set_print_timestamp(darktable.image_cache, params->imgid);
```

Comment:
I may be missing something... Where is this print_timestamp written into the db?

Reply:
It is written, like the other two in image_cache.c